### PR TITLE
feat - added list

### PR DESCRIPTION
**Pull request by ashishnnnnn**
-> Added Normal List
-> Added Stacked List .

### CODE REVIEW

File: list/list.html [hunk 1]
```diff
@@ -0,0 +1,316 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
```
Comment (arjun-computer-geek) [inline list/list.html:7]: It would be better if you put script tag in body and use defer attribute too

File: list/list.html [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+      <div class="left-nav mar-r-auto">
+        <a class="fnt-1-5 fnt-w-900" href="#">Pinakin</a>
+      </div>
+      <div class="right-nav">
+        <ul class="right-nav-list gap-1-5">
+          <li class="home">
+            <a href="../index.html"><i class="fnt-1-5 fas fa-home"></i></a>
+          </li>
+          <li class="github">
+            <a href="https://github.com/ashishnnnnn"
```
Comment (arjun-computer-geek) [inline list/list.html]: I  think you should use target="_blank" for external links.

File: list/list.html [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+              ><i class="fnt-1-5 fab fa-github"></i
+            ></a>
+          </li>
+          <li class="hamburger">
+            <input style="display: none" type="checkbox" id="click" />
+            <label for="click" class="menu-btn">
+              <i class="fas fa-bars"></i>
+              <!-- <i class="fas fa-times"></i> -->
```
Comment (arjun-computer-geek) [inline list/list.html]: Remove all unused commented codes. It reduces the readability.

**Review by arjun-computer-geek — CHANGES_REQUESTED** [review]
Please do changes what I mentioned it will be better or you can leave it depends on you. That was just a suggestion😁

File: list/list.html (hunk 1 above)
Comment (ashishnnnnn) [reply list/list.html:7]: This Script is for font-awesome..
And i want to load all the icon to load as soon the html rendering start..
That why i put it in head..

File: list/list.html (hunk 2 above)
Comment (ashishnnnnn) [reply list/list.html]: Done

File: list/list.html (hunk 3 above)
Comment (ashishnnnnn) [reply list/list.html]: Done :+1:

**Review by ishapatell1 — APPROVED** [review]
Good enough to merge with the dev branch.

**Review by shubhamkumar648 — COMMENTED** [review]
code looks good! to me!  suggestion :- you could also put comment for easy to readd code

**Pull request merged**